scripts/pythonscripts/sync_postgres_to_mssql.py

```diff
@@ -0,0 +1,228 @@
+import psycopg2
+import pyodbc
+import pandas as pd
+import numpy as np
+from concurrent.futures import ThreadPoolExecutor
+from typing import List
+import time
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+def get_schema_tables():
```

Comment:
Please add docstrings to each function. Take a look at some examples here: https://github.com/Century-Health/chealth-kedro-project/blob/main/src/chealth_kedro_project/pipelines/data_manager/nodes.py